### ISSUE TITLE
Something in framerate broke astra

### ISSUE THREAD

**Issue by rayrobdod** [rayrobdod/FEMultiPlayer-V2#1]
Issue type: `problem`
#### Description:

The attacker was Guy with a Brave Sword and the defender was a second Guy with Aldhumla standing on a Mountain. The Attacking Swordmaster activated Astra. The first two astra hits missed, and both clients crashed simultaneously upon the third astra hit. I believe that one hit, even an astra hit, would have managed to kill the defender.

Upon further investigation, it appears to be any instance of Astra will crash in c89a7bac43ca1fef4ee6eef2559acb619deb946d (framerate). Regardless of weapon, regardless of hit rates or hit points, and always on the third part of astra.

unable to reproduce in 9e6d973c9e8858b00ce91d03ed59301953f81261  (master)

---
#### Examples:

```
java.lang.IllegalArgumentException: Invalid state transit: 9 to 2
at net.fe.fightStage.FightStage.setCurrentEvent(FightStage.java:580)
at net.fe.fightStage.anim.NormalAttack.onLastHit(NormalAttack.java:32)
at net.fe.fightStage.anim.AttackAnimation.increment(AttackAnimation.java:101)
at chu.engine.anim.Animation.update(Animation.java:192)
at chu.engine.anim.Sprite.update(Sprite.java:170)
at chu.engine.Entity.onStep(Entity.java:76)
at net.fe.fightStage.FightStage.onStep(FightStage.java:292)
at net.fe.FEMultiplayer.loop(FEMultiplayer.java:331)
at net.fe.FEMultiplayer.main(FEMultiplayer.java:104)
```

---

```
[BATL]0 BATTLESTART::
[BATL]0 HP::Mia HP18 Guy HP6
[BATL]0 ATTACKRECORD::Attack Miss: Mia, Guy, 0, 0 (drain)
[BATL]0 ATTACKRECORD::Attack Critical(a): Guy, Mia, 18, 0 (drain)
[BATL]0 HP::Mia HP0 Guy HP6
[BATL]0 BATTLEEND::
[SEND]2 COMMANDMESSAGE::Mia move (0, 0):[EQUIP, Mia, 0, ATTACK, Guy]
[RCVE]2 COMMANDMESSAGE::Guy move (-2, 0):[EQUIP, Guy, 0, ATTACK, Guy]
[BATL]0 BATTLESTART::
[BATL]0 HP::Guy HP33 Guy HP6
[BATL]0 ATTACKRECORD::Astra1(a) Miss: Guy, Guy, 0, 0 (drain)
[BATL]0 ATTACKRECORD::Astra2(a) Miss: Guy, Guy, 0, 0 (drain)
[BATL]0 ATTACKRECORD::Astra3(a) Miss: Guy, Guy, 0, 0 (drain)
[BATL]0 ATTACKRECORD::Astra4(a) Miss: Guy, Guy, 0, 0 (drain)
[BATL]0 ATTACKRECORD::Astra5(a) Miss: Guy, Guy, 0, 0 (drain)
[BATL]0 ATTACKRECORD::Attack Miss: Guy, Guy, 0, 0 (drain)
[BATL]0 ATTACKRECORD::Attack: Guy, Guy, 20, 0 (drain)
[BATL]0 HP::Guy HP13 Guy HP6
[BATL]0 BATTLEEND::
[SEND]2 COMMANDMESSAGE::Guy move (-2, 0):[EQUIP, Guy, 0, ATTACK, Guy]
```

**Comment by rayrobdod** [comment]
Lyn's Astra is perfectly fine.

**Comment by rayrobdod** [comment]
Fixed by 660a92c79c66b83f4acecd02925fc108c04312a1
